src/services/speaking-timer.js

```diff
@@ -0,0 +1,39 @@
+let audioDuration = 0;
+let timeSeconds;
+let timer;
+
+const speakingTimerHandler = (timerOn) => {
+  const speakingTimerDomElement = document.getElementById("speaking-timer");
+  timeSeconds = 299;
+  if (timerOn === false) {
+    return stopTimer(speakingTimerDomElement);
+  } else {
+    startTimer(speakingTimerDomElement);
+  }
+};
+
+const stopTimer = (element) => {
+  clearInterval(timer);
+  element.textContent = "5:00";
+  return audioDuration;
+};
+
+const startTimer = (element) => {
+  audioDuration = 0;
+  timer = setInterval(() => {
+    let seconds = timeSeconds % 60;
+    const minutes = (timeSeconds / 60) % 60;
+    if (timeSeconds < 0) {
+      stopTimer(element);
+    } else {
+      if (seconds < 10) {
+        seconds = "0" + seconds;
+      }
+      ++audioDuration;
+      --timeSeconds;
+      element.textContent = `${Math.floor(minutes)}:${seconds}`;
+    }
+  }, 1000);
+};
```

Comment:
Check out the solution for timer I have:
![image](https://user-images.githubusercontent.com/60318924/127152661-649050de-06eb-4286-8ca9-9088bcb68b5c.png)

Reply:
it is not possible to read